library/include/G4CMPInterValleyScattering.hh

```diff
@@ -32,6 +32,12 @@ public:
   // Do scattering action here
   virtual G4VParticleChange* PostStepDoIt(const G4Track&, const G4Step&);
 
+  // Deprecated version of IV scattering
+  virtual G4VParticleChange* SwitchValleys(const G4Track&, const G4Step&);
+
+  // Up-to-date version of IV scattering
+  virtual G4VParticleChange* ValleyScattering(const G4Track&, const G4Step&);
```

Comment:
These don't have to be virtual (I don't think we want a future subclass to overload them).  But it doesn't cost anything, so it's not worth changing.